### DISCUSSION TITLE
Failure on GFLW initialization

### DISCUSSION THREAD

**Discussion by sagraziela** [codam-coding-college/MLX42#118] [Q&A]
I'm trying to run my fdf projetc in my personal notebook (WSL), but it doesn't work. Although it compiles, it does not open the window when executing the 'a.out' file, and after a while, it just returns the following error message on the terminal: 

"Failed to initialize GLFW".

What should I do to fix this error?

**Comment by W2Wizard** [comment]
Hello! Sorry for the delay of a response. Are you using WSL2? This error indicates something is wrong / missing in your environment, so this might be a display manager or old version of wsl (wsl1 for instance cannot handle window management)
